tests/test_project.py

```diff
@@ -1941,8 +1941,10 @@ def do_run(self, args, ignored):
 
 
 def test_extension_command_duplicate(repos_tmpdir):
-    # Test to ensure that in case to subprojects introduces same command, it
-    # will print a warning.
+    # West should disregard an extension command if its name is already taken,
+    # either by a built-in command or another extension command added earlier.
+    # A warning should also appear for each such case.
```

Comment:
pydoc syntax instead?

Reply:
Do you mean: move this comment above the function, or make it a docstring?
So far, I just kept it consistent with other comments in this particular file.

Reply:
>  I can't see a good reason why.

I tried to write pydoc for APIs and comments for things that weren't API, as a pattern.

Reply:
Thanks @mbolivar , now I see https://docs.zephyrproject.org/latest/develop/west/west-apis.html is extracted from the source. Is that automated somewhere or manual?

> I tried to write pydoc for APIs and comments for things that weren't API, as a pattern.

Makes sense but the `tests/` are located in an entirely different, top-level directory so they should not interfere.

Either way this should really be written down somewhere.

Apologies @57300 for the off-topic discussion, feel free to ignore.